monitoring/snippets/v3/uptime-check-client/snippets_test.py

```diff
@@ -87,19 +87,19 @@ def call_sample():
         assert new_uptime_check_path in out
```

Comment:
Return a result from the tested code and assert something about the return value, not captured text.